recipes/pygeohash/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pygeohash" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 750e51643e743eabd065a84fc8c1912c5843b648143137919e4a9776366d921e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  files:
+    - tests/test_pygeohash.py
+  imports:
+    - pygeohash
+
+about:
+  home: https://github.com/wdm0006/pygeohash
+  license: AGPL-3.0
+  license_family: AGPL
+  license_file: LICENSE.txt
+  summary: 'A module for decoding and encoding geohashes to and from latitude and longitude coordinates'
+
+  # The remaining entries in this section are optional, but recommended.
+  description: |
+    Pygeohash is a Python module that provides functions for
+    decoding and encoding geohashes to and from latitude and
+    longitude coordinates, and doing basic calculations and
+    approximations with them.
+
+
+extra:
+  recipe-maintainers:
+    - adbeda
```

Comment:
You need an extra line here to make the linter happy

Reply:
Got it. I noticed that when I looked into why it failed.

Reply:
@chrisburr, this is my first contribution, what's the next step on this? should I go ahead and create another pull request after fixing the errors ?

Reply:
No need for another PR, just push to this branch again and the tests will run again.